Mathlib/LinearAlgebra/Basic.lean

```diff
@@ -546,10 +547,12 @@ def smulOfUnit (a : Rˣ) : M ≃ₗ[R] M :=
 /-- A linear isomorphism between the domains and codomains of two spaces of linear maps gives a
 linear isomorphism between the two function spaces. -/
 def arrowCongr {R M₁ M₂ M₂₁ M₂₂ : Sort _} [CommSemiring R] [AddCommMonoid M₁] [AddCommMonoid M₂]
```

Comment:
```suggestion
def arrowCongr {R M₁ M₂ M₂₁ M₂₂ : Type(} [CommSemiring R] [AddCommMonoid M₁] [AddCommMonoid M₂]
```
Can you repeat `R₁` here too? I think it's hard to tell what's going on when one variable is way higher up the file, and all the rest are right here.

Reply:
I'm assuming you meant `Type*` here?